great_expectations/datasource/data_connector/configured_asset_sql_data_connector.py

```diff
@@ -60,13 +62,37 @@ def assets(self) -> Dict[str, dict]:
 
     def add_data_asset(
         self,
-        name: str,
-        config: dict,
+        data_asset_name: str,
+        data_asset_config: dict,
```

Comment:
This function is only called using unnamed-parameters within the codebase, so the name change does not have a breaking effect. It can be switched back if reviewers would like.

Reply:
I don't feel particularly strongly either way although I think it is somewhat implied we're supplying `data_asset_name` and `data_asset_config` because of the context around the method.

Reply:
I'm with @cdkini. I think the original way was more consistent with the rest of the codebase as well.